.github/workflows/test.yml

```
@@ -78,56 +74,48 @@ jobs:
       - name: Fetch git tags
         run: git fetch origin 'refs/tags/*:refs/tags/*'
 
-      - name: Setup caching for conda packages
-        uses: actions/cache@v2
+      - name: Setup Python
+        uses: actions/setup-python@v2
         with:
-          path: ~/conda_pkgs_dir
-          key: conda-${{ runner.os }}-${{ matrix.python }}-${{ hashFiles('requirements*.txt') }}
+          python-version: ${{ matrix.python }}
 
-      - name: Setup Miniconda
-        uses: conda-incubator/setup-miniconda@v2.0.1
+      - name: Get the pip cache folder
+        id: pip-cache
+        run: |
+          echo "::set-output name=dir::$(pip cache dir)"
+
+      - name: Setup caching for pip packages
+        uses: actions/cache@v2
         with:
-          python-version: ${{ matrix.python }}
-          miniconda-version: "latest"
-          auto-update-conda: true
-          channels: conda-forge
-          show-channel-urls: true
-          activate-environment: testing
-          # Needed for caching
-          use-only-tar-bz2: true
+          path: ${{ steps.pip-cache.outputs.dir }}
+          key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements*.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-pip-
```

```suggestion
            ${{ runner.os }}-pip-${{ hashFiles('**/requirements*.txt') }}
```

Seems reasonable right? That way we restore the cache if the requirements haven't changed?

---

Yes! Thanks for the catch!